src/main/java/io/codelex/flightplanner/admin/AdminRepository.java

```diff
@@ -0,0 +1,55 @@
+package io.codelex.flightplanner.admin;
+
+import io.codelex.flightplanner.admin.domain.Airport;
+import io.codelex.flightplanner.admin.domain.Flight;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Repository
+public class AdminRepository {
```

Comment:
Ja AdminRepository un AdminService tiek izmantots arī iekš CustomerService, vai šis joprojām ir tiesīgs saukties AdminRepository? :)
Faktiski ir tā, ka controller mums var būt dažādi - Customer, Admin, Testing u.t.t., bet service un repository būtu jāatbilst domain objektiem - jebšu faktiski Flight (teorētiski varētu arī Airport izdalīt, bet to šoreiz var arī nedarīt). Mans ieteikums - uztaisi vienkārši FlightService un tajā saliec kodu no AdminService, CustomerService un TestingService, un izmanto šo FlightService visos Controller.
 Un ar repository tas pats - Uztaisi flight repository (vai faktiski pārdēvē šo par FlightRepository, jo šī klase jau nekādus Admin nesaglabā, bet gan darbojas ar Flight. Ja par šo vēl jautājumi - droši uzraksti.